bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -47,16 +53,65 @@ public function __construct(
         GsOrderRepository $gsOrderRepository,
         KernelInterface $appKernel,
         MessageBusInterface $messageBus,
-        string $ownCloudFilesFolder
+        string $ownCloudFilesFolder,
+        TokenStorageInterface $tokenStorage,
+        FileUploader $fileUploader,
+        UserManager $userManager
     ) {
         $this->entityManager = $entityManager;
-        $this->gsOrderRepository = $gsOrderRepository;
         $this->hammerService = $hammerService;
         $this->ownCloudService = $ownCloudService;
         $this->gsOrderRepository = $gsOrderRepository;
         $this->appKernel = $appKernel;
         $this->messageBus = $messageBus;
         $this->ownCloudFilesFolder = $ownCloudFilesFolder;
+        $this->tokenStorage = $tokenStorage;
+        $this->fileUploader = $fileUploader;
+        $this->userManager = $userManager;
```

Comment:
Missing in private variable declarations in the top of the class.

Reply:
fixed